mscore/svggenerator.cpp

```diff
@@ -1126,10 +1129,28 @@ void SvgPaintEngine::updateState(const QPaintEngineState &state)
     if (!qFuzzyIsNull(state.opacity() - 1))
         stateStream << SVG_OPACITY << state.opacity() << SVG_QUOTE;
 
-    // Set these class variables for later use in the drawXXX() functions
-    QMatrix mx = state.matrix();
-    _dx = mx.dx();
-    _dy = mx.dy();
+    // Translations, SVG transform="translate()", are handled separately from
+    // other transformations such as rotation. Qt translates everything, but
+    // other transformations do occur, and must be handled here.
+    QTransform t = state.transform();
+    if (t.m11() == t.m22()    // No scaling
+     && t.m12() == t.m21()) { // No rotation, etc.
+        // No transformation except translation
```

Comment:
style: we indent by 6 spaces, not by 4, see https://musescore.org/en/developers-handbook/musescore-coding-rules#Indentation-%26-Whitespaces, , the XML at the ebd of that page helps if you're using QT-Creator